Update starkgate-estimating_fees.adoc (#1101)
starknet-io/starknet-docs: components/Starknet/modules/tools/pages/starkgate-estimating_fees.adoc

[id="starkgate_estimating_fees"]
= Estimating StarkGate fees

> StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. For more information, see xref:architecture_and_concepts:Network_Architecture/messaging-mechanism[L1 -> L2 message].

Reviewer: ```suggestion
StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. For more information, see xref:architecture_and_concepts:Network_Architecture/messaging-mechanism.adoc#l1-l2-message-fees[L1 -> L2 message fees].
```

> https://starknet-io.github.io/starknet-docs/pr-1101/documentation/architecture_and_concepts/Network_Architecture/messaging-mechanism/

Reviewer: ```suggestion
```

[id="starkgate_estimating_fees"]
= Estimating StarkGate fees

> StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. For more information, see xref:architecture_and_concepts:Network_Architecture/messaging-mechanism[L1 -> L2 message].

Reviewer: ```suggestion
StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. For more information, see xref:architecture_and_concepts:Network_Architecture/messaging-mechanism[L1 -> L2 message].

https://starknet-io.github.io/starknet-docs/pr-1101/documentation/architecture_and_concepts/Network_Architecture/messaging-mechanism/
```

> StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. Check out the xref:https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/messaging-mechanism/[L1 -> L2 message] section in the documentation for more details.

Reviewer: ```suggestion
StarkGate enforces a minimum fee for all transactions to account for the L1 -> L2 message costs. For more information, see xref:architecture_and_concepts:Network_Architecture/messaging-mechanism[L1 -> L2 message].
```
